ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -177,6 +177,11 @@ open class BadgeView: UIView {
             updateBackgroundColor()
             updateLabelTextColor()
             isUserInteractionEnabled = isActive
+            if isActive {
+                accessibilityTraits.remove(.notEnabled)
+            } else {
+                accessibilityTraits.insert(.notEnabled)
+            }
```

Comment:
Could we put this inside something like "updateAccessibilityTraits" ?

Reply:
i didn't see much benefit in putting in a separate function for now. only one place should call it anyway.